test/e2e/pods.go

```diff
@@ -466,6 +469,116 @@ var _ = Describe("Pods", func() {
 		Logf("Pod update OK")
 	})
 
+	It("should have proper update count during pod creation and deletion", func() {
+		// This e2e test is mainly aimed at understand the update count for pod creation and deletion
+		// TODO(random-liu): Limit the update count later when we have a reasonable upper bound.
+		podClient := framework.Client.Pods(framework.Namespace.Name)
+
+		By("creating the pod")
+		name := "pod-update-" + string(util.NewUUID())
+		value := strconv.Itoa(time.Now().Nanosecond())
+		pod := &api.Pod{
+			ObjectMeta: api.ObjectMeta{
+				Name: name,
+				Labels: map[string]string{
+					"name": name,
+					"time": value,
+				},
+			},
+			Spec: api.PodSpec{
```

Comment:
ACK...I just use the same pod spec with the other test, but I think I should simplify it here. Thanks.